.github/workflows/build-server.yml

```
@@ -28,28 +31,32 @@ jobs:
         run: sudo snap install golangci-lint --classic
 
       - name: Run linter
-        run: ./run_linter.sh
+        run: make lint
+
+      - name: Install gotestsum
+        run: go install gotest.tools/gotestsum@latest
 
       - name: Test
         id: test
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:2:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)